testsuite/galleon/update/pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wildfly</groupId>
+        <artifactId>wildfly-ts-galleon-parent</artifactId>
+        <!--
+        Maintain separation between the artifact id and the version to help prevent
+        merge conflicts between commits changing the GA and those changing the V.
+        -->
+        <version>27.0.0.Alpha2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>wildfly-ts-galleon-update</artifactId>
+    <packaging>pom</packaging>
+    <name>WildFly Test Suite: Galleon update tests</name>
+    <description>Install a base version and check that wildfly producers can get updated to the SNAPSHOT version.</description>
+    <profiles>
+        <profile>
+            <id>galleon.profile</id>
+            <activation>
+                <os>
+                    <family>unix</family>
+                </os>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-enforcer-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>enforce-property</id>
+                                <goals>
+                                    <goal>enforce</goal>
+                                </goals>
+                                <configuration>
+                                    <rules>
+                                        <requireProperty>
+                                            <property>wildfly.test.galleon.update.base.version</property>
+                                            <message>You must set wildfly.test.galleon.update.base.version property to a WildFly version that will be used to install prior to update to the latest SNAPSHOT!</message>
```

Comment:
Setting this via a maven property in the root pom seems better. That can be overridden from the command line, but it provides a single place to update this when the 'default' value changes, instead of having to edit various CI jobs.

Reply:
@bstansberry , thank-you. I added a default value, 26.1.1.Final for now.